05_deploy_bootstrap_vm.sh

```diff
@@ -117,15 +117,21 @@ sudo systemctl reload NetworkManager
 cat /tmp/dnsmasq.conf | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=dnsmasq.conf
 
 # Build and start the ironic container
-cat ironic/runironic.sh | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=runironic.sh
-cat ironic/Dockerfile | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=Dockerfile
-ssh -o "StrictHostKeyChecking=no" core@$IP sudo podman build \
-    --build-arg RHCOS_IMAGE_URL=${RHCOS_IMAGE_URL} \
-    --build-arg RHCOS_IMAGE_VERSION=${RHCOS_IMAGE_VERSION} \
-    --build-arg RHCOS_IMAGE_FILENAME_OPENSTACK=${RHCOS_IMAGE_FILENAME_OPENSTACK} \
-    -t ironic:latest .
+cat "${RHCOS_IMAGE_FILENAME_OPENSTACK}" | ssh -o "StrictHostKeyChecking=no" "core@$IP" sudo dd of="${RHCOS_IMAGE_FILENAME_OPENSTACK}"
+
+IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metalkube/metalkube-ironic"}
+if [ -z ${IRONIC_IMAGE_REPO:-} ]; then 
+  ssh -o "StrictHostKeyChecking=no" "core@$IP" git clone "${IRONIC_IMAGE_REPO}"
+  IRONIC_IMAGE="localhost/stack/metalkube-ironic"
+  ssh -o "StrictHostKeyChecking=no" "core@$IP" "cd ${IRONIC_IMAGE_REPO}; podman build -t ${IRONIC_IMAGE} ."
```

Comment:
Looks like $IRONIC_IMAGE_REPO is a git repository, we'd need to cd into the directory name.

Reply:
Had to remove that part - RHCOS image doesn't have git, so either the image needs to be uploaded to some registry or built manually on the boostrap node